src/common/readme/assets/README.md

```diff
@@ -0,0 +1,47 @@
+# {{PROJECT_TITLE}}
+
+The app is created using a [projen](https://projen.io) template.
+
+## Update a project
+
+In order to pull template updates you need to specify the desired version of the `@ottofeller/templates` package in either `.projenrc.ts` file or in `package.json` (if the version in `.projenrc.ts` is not fixed). Then run the default projen task:
+
+```sh
+npx projen
+```
+
+Upon completion the following changes would apply:
+
+- all packages with specified versions are updated (if a template does not specify a dependency version, it is not managed by projen and can be updated by simply setting the desired version in `package.json`; note that this way `package-lock.json` is not necessarily updated, thus you need to check it as well);
+- all projen-generated files are updated;
+- all sample code files are left unchanged (note that if you have deleted some sample code files they will be recreated unless you use the `sampleCode: false` option).
+
+## Install new packages
+
+The common approach of installing packages by running `npm install <package-name>` won't work because `npx projen` re-synthesizes all files, including `package.json`. For dependency handling see the docs in `projen` repo:
+
+- [generic dependency handling](https://github.com/projen/projen/blob/main/docs/deps.md);
+- [node.js specific](https://github.com/projen/projen/blob/main/docs/node.md#dependencies);
+- [node-package comments on dependency options](https://github.com/projen/projen/blob/main/src/javascript/node-package.ts#L46-L111) - these are available in an IDE.
+
+To install a new packages to the project:
+
+- Add a new item with the package name to either the `deps` or `devDeps` array in project options. Alternative way would be to use `project.addDeps('package-name')` or `project.addDevDeps('dev-package-name')`.
+- Run `npx projen`. This will update the `package.json` and lock file as well.
+
+Note that there are two distinct approaches in controlling package version:
```

Comment:
@quesabe let's try to keep the doc as short as we can, so that people can quickly find answers to the most important questions. I would only keep the practical parts of the documentation here. For the rationale of those we should refer links in our docs I think.

Reply:
Ok.

However, considering the repo is public, is it relevant to post links to closed-source docs here? Though I guess It's OK to link to the templates repo README, as the readme of a project is just a concise extract from the templates readme.

Reply:
But I guess you a referencing parts of the [`ottofeller/tempplates`](https://github.com/ottofeller/templates/?tab=readme-ov-file#install-new-packages) repo, which is public?